dot/rpc/modules/childstate.go

```diff
@@ -29,6 +29,13 @@ type GetKeysRequest struct {
 	Hash   common.Hash
 }
 
+// GetStorageHash the request to get the entry child storage hash
+type GetStorageHash struct {
+	KeyChild []byte
+	EntryKey []byte
+	Hash     common.Hash
```

Comment:
since block hash is optional, can you make this `*common.Hash`?

Reply:
Done!